developer/build_plugin.py

```diff
@@ -143,32 +143,66 @@ def _process_configuration(sg_connection, source_path, target_path, bundle_cache
                 "version": BAKED_BUNDLE_VERSION
             }
         )
-
+        # Workaround for tk-core bootstrap needing a shotgun.yml file: create
```

Comment:
Maybe we could add a bit more info here than just workaround? Thinking about the case when we revisit this code in the future and wonder 'what was the backstory here?' - if we could add a bit more info about why we are doing this, i think that could be helpful.

Reply:
is this workaround needed for all plugins or just baked ones?